src/protocol/PasLS.InactiveRegions.pas

```diff
@@ -0,0 +1,140 @@
+// Pascal Language Server
```

Comment:
This should be your name and 2023 right?

Reply:
Done